libs/net/dal/Migrations/Initial/Up/PostUp/01-DataSource.sql

```diff
@@ -2916,20 +2916,44 @@ INSERT INTO public.data_source (
   , ''
 ), (
   'CBCV'
-  , 'CBCV'
+  , 'CBCV-CAPTURE'
   , ''
   , true -- is_enabled
-  , 1 -- content_type_id
+  , NULL -- content_type_id
   , 3 -- media_type_id
   , 2 -- data_location_id
   , 3 -- license_id
   , ''
-  , '{}' -- connection
+  , '{
+    "serviceType":"stream",
+    "url":"https://cbcrclinear-tor.akamaized.net/hls/live/2042769/geo_allow_ca/CBCRCLINEAR_TOR_15/master4.m3u8",
+    "timezone":"UTC",
+    "timeZone":"Pacific Standard Time"
+  }' -- connection
   , NULL -- parent_id
   , DEFAULT_USER_ID
   , ''
   , DEFAULT_USER_ID
   , ''
+), (
+  'CBCV-CLIP'
+  , 'CBCV'
+  , ''
+  , true -- is_enabled
+  , 1 -- content_type_id
+  , 3 -- media_type_id
+  , 2 -- data_location_id
+  , 3 -- license_id
+  , 'CBCV'
+  , '{"serviceType":"clip",
+    "keepChecking":true,
+    "timezone":"UTC",
+    "timeZone":"Pacific Standard Time"}' -- connection
+  , 182 -- parent_id
```

Comment:
It should be a query for the parent by its code.  If someone edits this file and adds records it'll be incorrect.